packages/sync/src/cache/createOptimisticResponse.ts

```diff
@@ -1,7 +1,7 @@
 
 // Helper method for ID generation ()
 const generateId = (length = 8) => {
-  let result = "";
+  let result = "client:";
```

Comment:
This will look better as  constant somewhere above.

Reply:
should be fixed now